_locales/en/messages.json

```diff
@@ -7,6 +7,10 @@
     "message": "Add Style",
     "description": "Title of the page for adding styles"
   },
+  "alphaChannel": {
+    "message": "Alpha channel",
+    "description": "Label of alpha channel input"
```

Comment:
I don't think every translator will know what an alpha channel is... it might be better to call it an opacity.

Reply:
The specification for rgba uses "alpha channel" so idk. Maybe `Alpha channel (opacity)` then?